frontend/src/components/TaskCreator/FocusPicker.tsx

```diff
@@ -27,7 +27,7 @@ export default function FocusPicker({ pinned, onPinChange }: Props): ReactElemen
         onKeyPress={pressedPicker}
         className={styles.Label}
       >
-        <SamwiseIcon iconName={pinIconName} className={styles.CenterIcon} />
+        <SamwiseIcon iconName={pinIconName} className={styles.CenterIcon} tabIndex={-1} />
```

Comment:
This is bad for accessibility.

Reply:
I did this so that the user can press the Tab button to move on from the task field to the subtask field. Do you have any other solution that lets me maintain this functionality without it tabbing to one of the icons instead?

Reply:
never mind, it seems that rearranging the elements to be in the intended tabbing order can help eliminate this attribute. thanks for the review!